### DISCUSSION TITLE
Tooltips on footnote numbers just like footnote reference ?

### DISCUSSION THREAD

**Discussion by UltraInstinct05** [squidfunk/mkdocs-material#4002] [Q&A] [answered]
Currently, footnote numbers don't have a tooltip https://squidfunk.github.io/mkdocs-material/reference/footnotes/#adding-footnote-references when hovered over. Maybe it could say `Jump to footnote <num>`

![image](https://user-images.githubusercontent.com/34304644/172288860-3fcffe42-72da-456d-bd25-1fffd49f0880.png)

But footnote references do have tooltips when hovered.

![image](https://user-images.githubusercontent.com/34304644/172288831-665863a1-fac8-4edd-bc30-53bbcea3649d.png)

Does it make sense to have tool tips on footnote numbers for consistency sake ?

**Comment by squidfunk** [comment, marked as answer]
Thanks for suggesting. Footnotes are rendered by the [footnotes](https://python-markdown.github.io/extensions/footnotes/) extension, which is an upstream dependency of ours. If footnotes would include a `title`, tooltips would work instantly. Thus, it might be worth raising this over at [Python Markdown](https://github.com/Python-Markdown/markdown).

**Reply by kvanderwijst** [reply]
Hi Martin, sorry to re-open this, but on the Mkdocs Material documentation itself, the footnote is shown as tooltip when hovering the footnote. This does not use `title` attributes as far as I can see. Is this an Insiders-functionality, or am I missing a configuration option? Thanks a lot!

<img width="565" alt="image" src="https://github.com/squidfunk/mkdocs-material/assets/863938/3a0bb775-788e-4452-99a6-da7a1e917db7">

**Reply by squidfunk** [reply]
Yes, it is an [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/) feature that we [released in January](https://squidfunk.github.io/mkdocs-material/insiders/changelog/#4.51.0) and called [footnote tooltips](https://squidfunk.github.io/mkdocs-material/reference/footnotes/#footnote-tooltips) ☺️ Sponsors can enable it with:

``` yaml
theme:
  features:
    - content.footnote.tooltips
```
